tests/playwright/resources/test-doc.html

```
@@ -0,0 +1 @@
+<html><head><title>Test</title></head><body><h1>Heading</h1><p>This is a test HTML document for Kaiden e2e tests.</p></body></html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)